queries.sql

```diff
@@ -0,0 +1,10 @@
+
+
+SELECT * FROM animals WHERE name like '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT * FROM animals WHERE neutered = 'true' AND  escape_attempts < 3;
```

Comment:
- [ ] Again here we are supposed to select the `name` column only not the entire record
![image](https://github.com/ragangithub/vet-clinic-db/assets/81932478/30af61dd-2507-4a5a-92f6-d5a50d6faf4e)
Check the requirements